UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -133,7 +133,28 @@ <h2>Introduction</h2>
 			<p>The good news is more and more publishers are creating publications that are Born Accessible (i.e.,
 				accessible from the outset, not fixed later) and getting the accessibility validation or audit done by
 				independent organizations.</p>
-		</section>
+		
+<section id="terminology">
+   <h3>Terminology</h3>
+   
+<p>There are several terms used in these guidelines that should   be defined  for clarity:</p>
+   <dl>
+      <dt><dfn>digital publication</dfn></dt>
+           <dd>
+         <p>A digital publication is a term that encompasses a wide range  of products  with a wide range of names. some examples are e-Books, audio books, manga, comic books, EPUB titles, PDF publications, journals, digital textbooks, picture books, children's picture books with accompanying audio, Digital Talking Books (DTB), and more.</p>
+      </dd>
+      <dt><dfn>Reading System</dfn></dt>
+<dd>
+<p>All digital publications require a Reading System to present the publication to the end user. Reading Systems may be Apps that run on a smart phone or tablet. There are Reading Systems which are  applications  that run on  Personal Computers. There are also Reading Systems that are integrated in to dedicated devices devoted to a single purpose, for presenting a publication. There are even skills   that run on Smart Speakers that can be considered Reading Systems.</p>
```

Comment:
We've moved away from capitalizing terms unless they also happen to be proper nouns. Reading system, apps, and personal computers should be lowercase here.